hijackers/config.go

```diff
@@ -0,0 +1,8 @@
+package hijackers
+
+import utls "github.com/refraction-networking/utls"
+
+type ClientTLsCredentials struct {
```

Comment:
Naming mistake: `s` in `TLS` should be uppercase too (here and in other usages too).